dissect/target/plugins/os/windows/wer.py

```diff
@@ -87,9 +87,14 @@ def _create_record_descriptor(record_name: str, record_fields: list[tuple[str, s
     return TargetRecordDescriptor(record_name, record_fields)
 
 
-def _key_to_snake_case(key: str) -> str:
+def _sanitize_key(key: str) -> str:
+    # Convert camel case to snake case
     for pattern in camel_case_patterns:
         key = pattern.sub(r"\1_\2", key)
+
+    # Keep only basic characters in key
+    key = re.sub(r"[^a-zA-Z0-9_]", "", key)
```

Comment:
I am not sure how important this is but I foresee two possibly unwanted results:

1. The key becomes empty ("невидимый")
2. Or you might end up with duplicate keys (áErreur, éErreur)*
* 'lowercasing' might also cause this

If that is no problem, then it does not matter of course.

Reply:
True! Changed the code a bit around to be able to warn the user when these edge cases happen. I chose for a warning, because I don't think it's worth it to fully handle these edge cases. But please let me know if you think differently about this.